src/structures/VoiceChannel.js

```diff
@@ -0,0 +1,14 @@
+const Base = require('./Base');
+
+module.exports = class VoiceChannel extends Base {
+  constructor(data) {
```

Comment:
Here, according to the coding style, we must put a space before the function's parentheses.
```suggestion
  constructor (data) {
```